components/server/about-page.tsx

```diff
@@ -36,9 +36,9 @@ const AboutPage: FC = (): ReactElement | null => {
           chess web-app.
         </AboutCard>
         <AboutCard imageName="default_picture.jpg" name="Simon Woidmann">
```

Comment:
Is in on purpose that there is no custom picture?

Reply:
Yeah, I could post a picture so that everyone has one.

Reply:
Up to you. Just wanted to make sure